### ISSUE TITLE
Non English Characters Briefly Appear as Node Label

### ISSUE THREAD

**Issue by pfsj** [codaco/Network-Canvas#399]
When loading a name generator, a node label briefly appeared in non-english language characters before changing to the correct characters. In my case, this specifically happened on the remote protocol for the node "Dee". However, I cannot replicate this bug. I will update the issue if I can figure out instructions to replicate the bug.

**Comment by jthrilly** [comment]
@pfsj - any repro steps for this?

**Comment by pfsj** [comment]
No. I both can't replicate this and I haven't seen it in quite a while. I'd say just close it down for now.
